crucible-llvm/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # next
 
+* The `doMallocHandle` function was removed.
+* The `bindLLVMFunPtr` function was removed, instead use `doResolveGlobal`
+  to resolve a global symbol to a pointer, `someFnHandle` to construct a
+  `SomeFnHandle`, and pass both to `doInstallHandle`.
```

Comment:
Completely reasonable, done! I think I just thought it was such a short function at that point it might not make sense, but I agree with you that it seems helpful, especially if it reduces churn downstream.